app/controllers/posts_controller.rb

```diff
@@ -1,12 +1,53 @@
 class PostsController < ApplicationController
+  before_action :find_post, only: [:like]
+
+  def like
+    @post.likes.create(author_id: current_user.id)
+    # Like.create(author_id: current_user.id, post_id: @post.id)
+    respond_to do |format|
```

Comment:
- [ ] Please consider removing commented codes as it will make your codebase messy, Remember if you ever like to see the codes you have written once you can use your commit history.

Reply:
Resolved accordingly @TadesseDev